config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml

```diff
@@ -60,6 +60,9 @@ presubmits:
           requests:
             cpu: 2
             memory: "9Gi"
+        env:
+          - name: API_SERVER_FEATURE_GATES
+            value: "UserNamespacesSupport=true"
```

Comment:
Does this mean this feature must be on when `ProcMountType` feature is enabled (which is now on by default)?  

that doesn't feel right to me.  Should we be fixing the validation logic that is causing it to fail?